components/Menu.tsx

```diff
@@ -0,0 +1,296 @@
+import Colors from "../constants/Colors";
+import CampusEventContainer from "./CampusEventContainer";
+import Building from "../classes/building";
+import { obtainBuildings } from "../services/buildingService";
+
+const styles = StyleSheet.create({
+  container: {
+    alignSelf: "flex-start",
+  },
+  iconSize: {
+    width: 30,
+    height: 30,
+  },
+
+  hamburger: {
+    position: "absolute",
+    zIndex: 9,
+    alignSelf: "flex-start",
+    top: 8,
+    left: 25,
+    margin: 5,
+  },
+  outsideModal: {
+    width: Dimensions.get("window").width,
+    height: Dimensions.get("window").height,
+    backgroundColor: Colors.black,
+    zIndex: 1,
+    opacity: 0.7,
+  },
+  modalContainer: {
+    width: Dimensions.get("window").width - 100,
+    height: Dimensions.get("window").height,
+    zIndex: 2,
+    position: "absolute",
+    backgroundColor: Colors.white,
+    padding: 10,
+    shadowColor: Colors.black,
+    shadowOffset: {
+      width: 0,
+      height: 2,
+    },
+    shadowOpacity: 0.25,
+    shadowRadius: 3.84,
+    elevation: 5,
+  },
+
+  modalEventSettingsContainer: {
+    justifyContent: "center",
+    alignItems: "center",
+    zIndex: 2,
+    top: 120,
+    width: 300,
+    height: 350,
+    position: "absolute",
+    alignSelf: "center",
+    backgroundColor: Colors.white,
+    padding: 10,
+    shadowColor: Colors.black,
+    shadowOffset: {
+      width: 0,
+      height: 2,
+    },
+    shadowOpacity: 0.25,
+    shadowRadius: 3.84,
+    elevation: 5,
+  },
+
+  modalLogo: {
+    height: 200,
+    width: "100%",
+    backgroundColor: Colors.menuBeige,
+    marginBottom: 10,
+    justifyContent: "center",
+    alignItems: "center",
+    padding: 15,
+  },
+  logoSize: {
+    alignSelf: "center",
+    marginBottom: 10,
+  },
+  syncBtn: {
+    backgroundColor: Colors.primaryColor,
+    height: 30,
+    width: "100%",
+    color: Colors.white,
+    justifyContent: "center",
+    alignItems: "center",
+  },
+  syncText: {
+    color: Colors.white,
+  },
+  menuOptions: {
+    width: "100%",
+    height: 40,
+    alignSelf: "flex-start",
+    flexDirection: "row",
+    padding: 5,
+    justifyContent: "flex-start",
+    alignItems: "center",
+    marginBottom: 10,
+  },
+  aboutLine: {
+    backgroundColor: Colors.grey,
+    width: "100%",
+    height: 1,
+    opacity: 0.4,
+  },
+  iconMenu: {
+    marginRight: 10,
+    width: 25,
+    height: 25,
+  },
+  pickContainer: {
+    marginTop: 15,
+    borderWidth: 0.4,
+  },
+  buildPicker: {
+    width: 250,
+    height: 30,
+  },
+});
+
+class Menu extends Component<
+  {},
+  {
+    showMenu: boolean;
+    showEvent: boolean;
+    showSettings: boolean;
+    buildings: Building[];
+    building: Building;
+  }
+> {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      showMenu: false,
+      showEvent: false,
+      showSettings: false,
+      buildings: obtainBuildings(),
+      building: new Building("", "", null, null, "", null, "H"),
```

Comment:
If you are initializing a null building, make sure the ending is also 'null' and not 'H'

Reply:
resolved

Reply:
resolved

Reply:
I will let this one to H since the first building in the picker is H. So by default it will show the first picker.